content/updates/2017-05-18-release-notes.md

```diff
@@ -0,0 +1,45 @@
++++
+date = "2017-05-18"
+title = "Platform Release Notes: May 18, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+### Added
+- We’ve published a cost estimator spreadsheet (in [XLSX](/resources/cloudgov_cost_estimator.xlsx) and [ODS](/resources/cloudgov_cost_estimator.ods) formats) that your team can use to get a sense of [how much cloud.gov will cost for your organization]({{< relref "overview/pricing/rates.md#find-the-right-package-for-your-work" >}}). If you’re interested in switching from a sandbox to a paid cloud.gov package, or if you want to expand your use of the platform to additional systems, use the estimator to get a sense of what the access package and usage quota fees will be across all your cloud.gov systems.
+- The Defense Information Systems Agency (DISA) has issued a [provisional authorization for Department of Defense teams]({{< relref "overview/security/fedramp-tracker.md#how-this-p-ato-helps-your-team" >}}) to use cloud.gov’s FedRAMP P-ATO for systems at the DISA level 2 impact level. This is a followup to the P-ATO; Level 2 is quivalent to FedRAMP the Moderate impact level.
+- The latest production releases for cloud.gov now support PHP 7.1.4 and 7.0.18.
+
+### Fixed
+Elastic Load Balancers (ELBs) now perform router health checks before putting a particular router back in service. Previously, ELBs occasionally produced 502 errors by sending traffic across unhealthy routers.
```

Comment:
We need to make this more customer-centric; assume they don't know what ELBs or routers are! How about:

> We've improved our process for updating the part of cloud.gov that routes traffic to your application. Previously, you might have seen occasional 502 errors when we made updates.

?